routes/vote/vote.controller.js

```diff
@@ -42,9 +54,15 @@ const postOptIn = (req, res, next) => {
   next()
 }
 
-const postConfirmRegistration = (req, res, next) => {
-  req.session.vote.voterCitizen = req.body.voterCitizen == 'voterCitizen' ? 'Yes' : 'No'
-  req.session.vote.voterConsent = req.body.voterConsent == 'voterConsent' ? 'Yes' : 'No'
+const postCitizen = (req, res, next) => {
+  const citizen = req.body.citizen
+  req.session.vote.citizen = citizen
 
+  if (citizen === 'Yes') {
+    return res.redirect('/vote/register')
```

Comment:
Small comment (don't know if it matters but what do you think):

- If you come back here from checkAnswers and then click "yes", you have to fill out the rest of the flow (see example)


![vote](https://user-images.githubusercontent.com/2454380/74376484-32cb7980-4db0-11ea-8346-0942deb5889e.gif)

It's not so many pages so it's not a big deal, but technically we should preserve the "checkAnswers" value.

Reply:
Agreed, I'll fix this.